recipes/Time_Series/Time_Series_Getting_Started.ipynb

```diff
@@ -429,7 +421,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Create a forecasting pipeline"
+    "### Create a forecasting pipeline\n",
+    "\n",
+    "Set up the forecasting pipeline with the model, setting `frequency` given our knowledge of the sample frequency. In this example we set `explode_forecasts` to `True`, which will format the output for plotting the history and prediction intervals. We then make a forecast on the dataset."
```

Comment:
Change: "output for plotting the history and prediction intervals" to "output for plotting the history and prediction period"